Formula/liquigraph.rb

```diff
@@ -0,0 +1,23 @@
+class Liquigraph < Formula
+  desc "Migration runner for Neo4j"
+  homepage "http://www.liquigraph.org"
+  url "https://github.com/fbiville/liquigraph/archive/liquigraph-1.0-RC3.tar.gz"
+  sha256 "9b616bd8d8798374d316decd7c1ef60882bf81f6627511af4e3c710143882c47"
+  head "https://github.com/fbiville/liquigraph.git"
+
+  depends_on "maven" => :build
+  depends_on :java => "1.7+"
+
+  def install
+    system "mvn", "-q", "clean", "package", "-DskipTests"
+    mkdir "binaries"
+    system "tar", "xzf", "liquigraph-cli/target/liquigraph-cli-bin.tar.gz", "-C", "binaries"
+    mv "binaries/liquigraph-cli/liquigraph.sh", "binaries/liquigraph-cli/liquigraph"
+    libexec.install "binaries/liquigraph-cli/liquigraph", "binaries/liquigraph-cli/liquigraph-cli.jar"
+    bin.install_symlink libexec/"liquigraph"
+  end
+
+  test do
+    system "#{bin}/liquigraph", "-h"
```

Comment:
Done! I'm a Ruby newbie, so let me know if my test can be written in a better way or if I'm missing something.